src/hooks/useInstitutionBranding.ts

```
@@ -0,0 +1,67 @@
+import { useEffect, useState, useRef } from 'react';
+import { useBranding } from './useBranding';
+import type { BrandingData } from '../store/brandingStore';
+
+/**
+ * Hook to fetch and apply institution branding before login
+ * Uses the institutionId from meta tag to fetch branding data
+ */
+export const useInstitutionBranding = (apiUrl: string, institutionId: string | null) => {
```

ESLint

[error] Replace `apiUrl:·string,·institutionId:·string·|·null` with `⏎··apiUrl:·string,⏎··institutionId:·string·|·null⏎`

(prettier/prettier)